docs/Android/allowed-apps.md

```diff
@@ -0,0 +1,76 @@
+---
+layout: page
+title: Intelligence SDK Allowed Apps
+hide:
+  #- navigation
+  - toc
+---
+
+## Intelligence SDK Allowed Apps
+
+The WS1 SDK Custom Settings KVP `IntelSDKAllowedApps` (Allowed Apps lists) is a JSON Array which can be used to control which apps can report DEX Data.
+- If the current app’s ID matches one of the IDs in the `IntelSDKAllowedApps` array, DEX data collection and transmission will be enabled for that app (assuming DEX is opted in). 
+- If the current app’s ID is not in the list, DEX data collection and transmission will be blocked for that app, even if DEX is otherwise enabled. 
+- If the `IntelSDKAllowedApps` key is missing or the array is empty, all apps will be allowed to transmit DEX data (default behavior).
+- The SDK could dynamically start or stop DEX data transmission based on changes to this list when `setSDKControlConfig` is called.
+    - If DEX is already opted in at the time the app has been allow listed, DEX data will start transmitting.
+    - If DEX is opted out at the time the app has been allow listed, DEX will remain off until it is opted in.
+
+Obtain your app ID from Omnissa Intelligence console. Please check [Omnissa Intelligence SDK Data for Apps](https://docs.omnissa.com/bundle/WS1Intelligence/page/IntelIntelligenceSDKApps.html) for more information.
+
+## How to Use
+To inject this configuration, use the API `setSDKControlConfig` offered through the `Crittercism` singleton.
```

Comment:
Can we also add something which talks about how UEM custom SDK settings can be fetched from UEM SDK via a code snippet? Either here or in a better location of your choice. 
It would be nice to show the developers the complete code from fetching the custom sdk settings and injecting it into IntelSDK via the new API with code comments.